### ISSUE TITLE
Unable to log in after upgrade from 5.5.6 to 5.6.0 (docker)

### ISSUE THREAD

**Issue by linucksrox** [ampache/ampache#3494]
## Description
After upgrading my docker instance from 5.5.6 to 5.6.0, I can get to the login screen and see the version as 5.6.0. When I type the wrong credentials, it tells me login failed. But when I type the correct credentials (I've tried with two different user accounts) it just refreshes the page with no messages or anything else. After that I'm still not authorized, for example clicking a bookmark to a playlist redirects me to the login page.

## Environment

* Ampache version: 5.6.0
* Web server + version: Docker
* Ampache runs behind Traefik v2 which has been working until this upgrade, so I suspect an issue on the Ampache side

### Client type

* Web

## Logs

```shell
2023-03-28T14:47:10+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key eaoch53r4usbpqprcu8peakiaf no data found
2023-03-28T14:47:10+00:00 [ampache] (Ampache\Module\System\Session) -> Writing to eaoch53r4usbpqprcu8peakiaf with expiration 1680018430
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Login\DefaultAction" for action "default"
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key eaoch53r4usbpqprcu8peakiaf no data found
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Session) -> Writing to eaoch53r4usbpqprcu8peakiaf with expiration 1680018441
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key 41esciagetjc5rii96l73nc7fo no data found
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Session) -> Session created: 41esciagetjc5rii96l73nc7fo
2023-03-28T14:47:21+00:00 [eric] (Ampache\Module\System\AutoUpdate) -> Checking latest version online...
2023-03-28T14:47:21+00:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /var/www/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(63)
2023-03-28T14:47:21+00:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /var/www/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(73)
2023-03-28T14:47:21+00:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /var/www/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(89)
2023-03-28T14:47:21+00:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /var/www/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(102)
2023-03-28T14:47:21+00:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /var/www/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(111)
2023-03-28T14:47:21+00:00 [eric] (Ampache\Module\System\AutoUpdate) -> Request error: cURL error 56: Received HTTP code 404 from proxy after CONNECT
2023-03-28T14:47:21+00:00 [eric] (Ampache\Module\System\Session) -> Writing to 41esciagetjc5rii96l73nc7fo with expiration 1680018441
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Index\ShowAction" for action "show"
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Session) -> Session created: 4fd59503cd781ba87ec4572bccd36ece
2023-03-28T14:47:21+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key dpcef8gmtnktcs3apf4uc4c9b6 no data found
2023-03-28T14:47:22+00:00 [ampache] (Ampache\Module\System\Session) -> Writing to dpcef8gmtnktcs3apf4uc4c9b6 with expiration 1680018442
2023-03-28T14:47:22+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:47:22+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:47:22+00:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Login\DefaultAction" for action "default"
2023-03-28T14:47:22+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key eaoch53r4usbpqprcu8peakiaf no data found
2023-03-28T14:47:22+00:00 [ampache] (Ampache\Module\System\Session) -> Writing to eaoch53r4usbpqprcu8peakiaf with expiration 1680018442
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Index\ShowAction" for action "show"
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Session) -> Session created: c9c15ad25ae08617eb7a69c78a8b126b
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key ob0v3rm18veijg1g9tibqv72ec no data found
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Session) -> Writing to ob0v3rm18veijg1g9tibqv72ec with expiration 1680018488
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Login\DefaultAction" for action "default"
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Session) -> Unable to read value from key mg3n6bvfjes2tb2qcp2hi4do55 no data found
2023-03-28T14:48:08+00:00 [ampache] (Ampache\Module\System\Session) -> Writing to mg3n6bvfjes2tb2qcp2hi4do55 with expiration 1680018488
```

**Comment by linucksrox** [comment]
I have been testing this different ways and I can't determine why it doesn't work on my main docker host. I tried a fresh install of 5.5.6 and ran through the installation, but was getting a similar db connection error by the end of the install wizard. It works fine on another host...
The craziest thing is that I tried checking the "remember me" box and was finally able to log in. Everything works, except I can't log out.
Is it me? Is it Ampache? I have no idea.
